src/QsCompiler/LanguageServer/Program.cs

```diff
@@ -177,13 +177,13 @@ private static void Log(object msg, string? logFile = null, bool stdout = false)
 
         internal static QsLanguageServer ConnectViaStdInOut(string? logFile = null)
         {
-            Log($"Connecting via stdin and stdout.", logFile, stdout: true);
+            Log($"Connecting via stdin and stdout.", logFile);
```

Comment:
I couldn't figure out why #1298 started logging to stdout instead of stderr, and it confuses Azure Notebooks (which expects JSON-RPC messages on stdout, not log messages), so I'd like to remove it

Reply:
I agree, I think all logging should go to stderr.